Google.Api.Generator/Generation/MethodDetails.cs

```diff
@@ -194,6 +195,7 @@ public Field(ServiceDetails svc, MessageDescriptor msg, string fieldName)
                 public Typ Typ { get; }
                 public bool IsRepeated { get; }
                 public bool Required { get; }
+                public bool IsWellKnownType { get; }
```

Comment:
I'd call this IsWrapperType rather than IsWellKnownType - we're not doing anything with Any, Duration, Timestamp etc are we?

Reply:
Done